modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -59,7 +60,24 @@ $ oc -n openshift-nmstate delete nmstate nmstate
 $ oc delete --all deployments --namespace=openshift-nmstate
 ----
 
-. Delete all the custom resource definition (CRD), such as `nmstates`, that exist in the `nmstate.io` namespace by running the following commands:
+. After you deleted the `nmstate` CR, remove the `nmstate-console-plugin` console plugin name from the `console.operator.openshift.io/cluster` CR.
++
+.. Use the `jq` CLI tool to store the position of the `nmstate-console-plugin` entry that exists among the list of enable plugins. The following command stores the index of the entry in an environment variable named `INDEX`:
++
+[source,terminal]
+----
+INDEX=$(oc get console.operator.openshift.io cluster -o json | jq -r '.spec.plugins | to_entries[] | select(.value == "nmstate-console-plugin") | .key')
+----
++
+.. Remove the `nmstate-console-plugin` entry from the `console.operator.openshift.io/cluster` CR by running the following patch command that uses the `oc` CLI:
```

Comment:
This is just my opinion but I feel like the end could be trimmed off for simplicity, since it's already clear that the command is for the oc CLI:
```suggestion
.. Remove the `nmstate-console-plugin` entry from the `console.operator.openshift.io/cluster` CR by running the following patch command:
```
```suggestion
.. Remove the `nmstate-console-plugin` entry from the `console.operator.openshift.io/cluster` CR by running the following patch command that uses the `oc` CLI:
```